pandas/io/formats/style.py

```diff
@@ -763,6 +784,41 @@ def set_table_styles(self, table_styles):
         self.table_styles = table_styles
         return self
 
+    def hide_index(self):
+        """
+        Hide any indices from rendering.
+
+        .. versionadded:: 0.20.2
+
+        Returns
+        -------
+        self : Styler
+        """
+        self.index_visible = False
+        return self
+
+    def hide_columns(self, subset):
+        """
+        Hide columns from rendering.
+
```

Comment:
this is a bit unusual for pandas, but I think it's fine here. It's kind of like `drop`, except it's hiding instead of dropping.